From 4cd75e6c576ba6edbc606028e492ee6f89e65c93 Mon Sep 17 00:00:00 2001
From: Jose Luis Blanco-Claraco <joseluisblancoc@gmail.com>
Date: Mon, 16 Sep 2024 22:41:44 +0200
Subject: [PATCH] Fix usage of DEB_HOST_MULTIARCH in d/rules

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 7cf68cc..abd17cf 100755
--- a/debian/rules
+++ b/debian/rules
@@ -57,7 +57,7 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/noetic/setup.sh" ]; then . "/opt/ros/noetic/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/ros-noetic-mp2p-icp//opt/ros/noetic/lib/
+	dh_shlibdeps -l$(CURDIR)/debian/ros-noetic-mp2p-icp//opt/ros/noetic/lib/:$(CURDIR)/debian/ros-noetic-mp2p-icp//opt/ros/noetic/lib/${DEB_HOST_MULTIARCH}
 
 override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.34.1

